src/content/docs/style-guide/writing-docs/processes-procedures/understand-edit-docs-site-structure.mdx

```diff
@@ -8,17 +8,16 @@ redirects:
 
 ## Terms [#terms]
-
 When talking about the docs site structure, sometimes people use different words for the same things. Below is a list of terms that can help us communicate about the docs site structure: 
 
-* **Nav files**: In the github docs site, there are yaml files under the `nav` folder that are used to determine the docs site structure that we display. This structure is exposed in the docs site left sidebar and when you view docs category views (also known as index views, like [this one](/docs/style-guide/quick-reference)). 
+* **Nav files**: In the github docs site, there are yaml files under the `nav` folder that are used to determine the docs site structure that we display. This structure is exposed in the docs site left sidebar and when you view docs category views (also known as auto-index pages, like [this one](/docs/style-guide/writing-strategies)). 
 * **Folder**: In this context, "folders" refers to the actual docs site folder structure (those folders and files in the `content` section). Referring to "folders" can be a helpful way to differentiate between the actual folder structure and the displayed structure that's set using the nav files. 
 * **Sidebar**: On public-facing docs, the sidebar is what is visible on the left hand side, showing the structure of that category of docs. The sidebar and index view are both determined by the structure set in the nav file. 
-* **Category** or **sub-category**: we use these words a bit interchangeably to refer to specific areas of the docs site. For example, [this view](/docs/using-new-relic/welcome-new-relic/get-started) shows a list of docs in a specific category, as does [this higher level category view](/docs/using-new-relic). In this doc, we'll often use the phrase "category view" to indicate a page load of a specific category of docs.  
-* **Auto-index pages**: This is another way, the more technical way, to refer to a category view of docs (for example, [this view](/docs/using-new-relic/welcome-new-relic/get-started)). "Auto-index page" and "category view" are used a bit interchangably in this doc, with "category view" being used as it's more informal and easy to parse. (Note that this use of "index" is a different use of "index" than the [authored index.mdx files](/docs/style-guide/article-templates/landing-page-template), which are used to display landing pages.)  
+* **Category** or **sub-category**: we use these words a bit interchangeably to refer to specific areas of the docs site. 
+* **Auto-index pages** (mostly deprecated): This is the more technical way to refer to a category view of docs (for example, [this view](/docs/style-guide/writing-strategies)). While these views are still functional, we should no longer link to them from the docs, hence why these views are considered deprecated.
```

Comment:
they won't be functional anymore and instead will be handled by the 404 page if no redirect is set for them

Reply:
ah I see.